eo-maven-plugin/src/test/java/org/eolang/maven/GmiMojoTest.java

```diff
@@ -67,7 +68,7 @@ void bigSlowTest() throws Exception {
             }
             program.append("[x y z] > foo\n");
         }
-        final XML graph = GmiMojoTest.toGraph(program.toString());
+        final XML graph = GmiMojoTest.toGraph(program.toString(), null);
```

Comment:
@OlesiaSub this is a terrible practice: passing as an argument

Reply:
@OlesiaSub why not `**`?

Reply:
@yegor256 fixed, I didn't really pay attention to that test as it's disabled, so I forgot to fix it afterwards